src/test/java/org/pesmypetcare/webservice/controller/petmanager/PetFreqTrainingRestControllerTest.java

```diff
@@ -0,0 +1,121 @@
+package org.pesmypetcare.webservice.controller.petmanager;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.entity.FreqTrainingEntity;
+import org.pesmypetcare.webservice.service.FreqTrainingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.BDDMockito.willDoNothing;
+import static org.mockito.BDDMockito.willReturn;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@ExtendWith(MockitoExtension.class)
+public class PetFreqTrainingRestControllerTest {
+    private static String jsonFreqTrainingEntity;
+    private static String jsonField;
+    private static FreqTrainingEntity freqTrainingEntity;
+    private static List<Map<String, Object>> freqTrainingEntityList;
+    private static String owner;
+    private static String petName;
+    private static String date;
+    private static String date2;
+    private static String urlBase;
+    private static String slash;
+    private static String lineBreak;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private FreqTrainingService service;
+
+    @BeforeAll
+    public static void setUp() {
+        lineBreak = "{\n";
+        slash = "/";
+        jsonFreqTrainingEntity = lineBreak
+            + "  \"freqTrainingValue\": 5.3\n" + "}";
+        jsonField = lineBreak
+            + "  \"value\": 7.5\n" + "} ";
+        freqTrainingEntity = new FreqTrainingEntity();
+        freqTrainingEntityList = new ArrayList<>();
+        owner = "Hola";
+        petName = "Muy Buenas";
+        date = "2018-10-22T00:47:00";
+        date2 = "2020-10-22T00:47:00";
+        urlBase = "/freqTraining";
+    }
+
+    @Test
+    public void createFreqTrainingShouldReturnStatusOk() throws Exception {
+        willDoNothing().given(service).createFreqTraining(anyString(), anyString(), anyString(), isA(FreqTrainingEntity.class));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 128).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5400494)